### DISCUSSION TITLE
Rate limit login attempts

### DISCUSSION THREAD

**Discussion by sinnbeck** [postalserver/postal#2445] [Help with using Postal] [answered]
I just noticed that several servers are trying to bruteforce login to my mail server. Is there any way to rate limit them, or block them from logging in? Or limit which IP's can access the server (except for setting this up in the server firewall)

**Comment by willpower232** [comment, marked as answer]
People have configured fail2ban in the past, have a look at #1187

**Reply by sinnbeck** [reply]
Ah brilliant! Thank you so much!
